app/models/rent.rb

```diff
@@ -0,0 +1,6 @@
+class Rent < ApplicationRecord
+  validates :init_date, presence: true
```

Comment:
`validates :init_date, :end_date, :user_id, :book_id, presence: true`
also you could use `user` or `user_id` (it makes no difference for me, use it as you want)